transcendence-app/src/shared/db/postgres/utils.ts

```diff
@@ -1,14 +1,16 @@
+import { Logger } from '@nestjs/common';
+import { DatabaseError } from 'pg';
 import { BaseEntity, Query, MappedQuery } from '../models';
 import { PostgresPool } from './postgresConnection.provider';
 
+const PostgresLogger = new Logger('Database');
+
 export const entityQueryMapper = (entity: Partial<BaseEntity>): MappedQuery => {
-  return Object.keys(entity).reduce<MappedQuery>(
+  return Object.entries(entity).reduce<MappedQuery>(
     (mappedQuery, [key, value], index) => {
-      if (value !== undefined) {
-        mappedQuery.cols.push(key);
-        mappedQuery.params.push(`$${index + 1}`);
-        mappedQuery.values.push(value);
-      }
```

Comment:
I've changed this because, according to the [docs](https://node-postgres.com/features/queries), if the value is undefined, it will be converted to null.

Reply:
Never mind. I'm reverting this as before because it is more specific, and we don't rely on edge case behavior.